Engines/Wine/Plugins/virtual desktop/script.js

```diff
@@ -1,15 +1,25 @@
-const Wine = include("engines.wine.engine.object");
-
+/* eslint-disable no-undef */
+include("engines.wine.engine.object");
+include("utils.functions.system.virtual_desktop");
 include("engines.wine.plugins.regedit");
+const Wine = include("engines.wine.engine.object");
```

Comment:
Can you please move includes with a `const` in the beginning to the top? This helps the readability. 
Afterwards I recommend adding an empty row between the normal includes and the engine extensions (i.e. the plugins and verbs).